scripts/test-install-zip.sh

```diff
@@ -56,15 +71,20 @@ rm -r build/packages/*/
 
 # Install packages and verify
 for zipFile in build/packages/*.zip; do
-  PACKAGE_NAME_VERSION=$(basename ${zipFile} .zip)
+  installAndVerifyPackage ${zipFile}
+done
 
-  elastic-package install -v --zip ${zipFile}
+# try to install one package without elastic-package stack shellinit
+unset ELASTIC_PACKAGE_KIBANA_HOST
+unset ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME
+unset ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD
+unset ELASTIC_PACKAGE_CA_CERT
 
-  # check that the package is installed
-  curl -s \
-    -u ${ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME}:${ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD} \
-    --cacert ${ELASTIC_PACKAGE_CA_CERT} \
-    -H 'content-type: application/json' \
-    -H 'kbn-xsrf: true' \
-    -f ${ELASTIC_PACKAGE_KIBANA_HOST}/api/fleet/epm/packages/${PACKAGE_NAME_VERSION} | grep -q '"status":"installed"'
-done
+export ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
+export ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
+export ELASTIC_PACKAGE_KIBANA_HOST=https://127.0.0.1:5601
+export ELASTIC_PACKAGE_CA_CERT=${HOME}/.elastic-package/profiles/default/certs/ca-cert.pem
+
+zipFile="build/packages/$(ls -rt build/packages/ | tail -n 1)"
+
+installAndVerifyPackage ${zipFile}
```

Comment:
With this, I tried to mimic what it would be installing a package in another Kibana instance.

Reply:
Could we keep testing both modes, maybe with a parameter on this script and two different tests?